scripts/load_testing.py

```
@@ -101,57 +102,57 @@ def remove_calculated_properties(data):
-                    raise Exception(f'PUT error {response.status_code}')
+                    raise Exception(f"PUT error {response.status_code}")
 
                 print(
-                    f'Updated {url} in execution {index+1} of thread ' \
-                    f'{(self.thread_number+1)}: Status {response.status_code}, ' \
-                    f'Time {response.elapsed.total_seconds()}'
+                    f"Updated {url} in execution {index+1} of thread "
+                    f"{(self.thread_number+1)}: Status {response.status_code}, "
+                    f"Time {response.elapsed.total_seconds()}"
                 )
             except Exception as exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)